test/models/customer_test.rb

```diff
@@ -1,4 +1,17 @@
 require "test_helper"
 
 class CustomerTest < ActiveSupport::TestCase
+  attr_reader :customer
+
+  def setup
+    @customer = customers(:one)
+  end
+
+    test "a customer has invoices" do
+    assert customer.methods.include?(:invoices)
+  end
```

Comment:
`end` at 12, 2 is not aligned with `test "a customer has invoices" do` at 10, 4.